lostfiles.py

```diff
@@ -158,12 +271,27 @@ def should_ignore_path(filepath: str) -> bool:
 
     return False
 
+def check_process(process_name: str) -> bool:
+    """
+    Check process is running based on name.
+    """
+    for proc in psutil.process_iter():
+        if proc.name() == process_name:
+            return True
+
+    return False
 
 def resolve_symlinks(*paths) -> Set[str]:
     return set(
         itertools.chain.from_iterable((path, os.path.realpath(path)) for path in paths)
     )
 
+def package_exist(name: str) -> bool:
+	for file in glob(PORTAGE_DB + "/" + name + "-[1-9]*"):
```

Comment:
Just a minor thing out of personal preference, this could be rewritten as `f"{PORTAGE_DB}/{name}-[1-9]*"`

Reply:
Fix it

Reply:
Sorry I wasn't clear on this one, the `glob()` call is still needed, I was referring to the string concatenation, which got improved in recent python versions: https://docs.python.org/3/tutorial/inputoutput.html#fancier-output-formatting.

It boils down to:
```
for file in glob(f"{PORTAGE_DB}/{name}-[1-9]*"):
    ...
```